x-pack/plugins/actions/server/create_execute_function.test.ts

```diff
@@ -68,6 +69,68 @@ describe('execute()', () => {
     });
   });
 
+  test('schedules the action with all given parameters with a preconfigured action', async () => {
```

Comment:
This is a copy of the test above it, only changing to use a preconfigured action.